app/assets/stylesheets/accountants.scss

```diff
@@ -0,0 +1,9 @@
+// Place all the styles related to the dashboards controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+#search_form {
```

Comment:
My inclination is to do this with bootstrap `col-sm-X` classes rather than css? Makes it a little more discoverable imo. Yr thoughts?